lib/model.js

```diff
@@ -174,23 +188,6 @@ function findMatchingTokens(token) {
  *   when fulfilled
  * @return Function
  */
-function getDocument(p_cursor) {
-  return function() {
-    return p_cursor.then(function(cursor) {
-      logger.debug('Retrieving next document from cursor');
-      return cursor.next();
-    });
-  };
-}
-
-/**
- * Checks that a cursor only contains one result. Returned promise rejects if
- * there was no results or more than one.
- *
- * @param {Promise} p_cursor - takes value of a mongodb.Cursor when fulfilled.
- * @return {Promise} Fulfills if only one token exists in the db, otherwise
- *   rejects with a {@link DbError}.
- */
 function cursorHasExactlyOneDoc(p_cursor) {
```

Comment:
cursorHasExactlyOneDoc has the jsdoc for getDocument, and cursorHasExactlyOneDoc's jsdoc seems to have been deleted